test/schedule/minischedule/load_test.exs

```diff
@@ -5,39 +5,32 @@ defmodule Schedule.Minischedule.LoadTest do
   alias Schedule.Hastus.Activity
   alias Schedule.Hastus.Trip
   alias Schedule.AsDirected
-  alias Schedule.Minischedule.Block
   alias Schedule.Break
   alias Schedule.Minischedule.Load
   alias Schedule.Piece
   alias Schedule.Minischedule.Run
 
-  describe "from_hastus/3" do
-    test "trips become pieces in run and block" do
+  describe "runs_from_hastus/3" do
```

Comment:
Should probably also add a test for `blocks_from_runs` (unless that simply goes away in a future commit).

Reply:
it does disappear